Containers/fulltextsearch/Dockerfile

```diff
@@ -3,4 +3,12 @@ FROM elasticsearch:7.17.8
 
 RUN elasticsearch-plugin install --batch ingest-attachment
 
-HEALTHCHECK CMD curl -skfI localhost:9200 || exit 1
+RUN set -ex; \
+    \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        tzdata \
+    ; \
+    rm -rf /var/lib/apt/lists/*
+
+HEALTHCHECK CMD nc -z localhost 9200 || exit 1

```

Comment:
are you sure that nc is included by default?

Reply:
```
docker run --rm -it --entrypoint nc elasticsearch:7.17.8
Unable to find image 'elasticsearch:7.17.8' locally
7.17.8: Pulling from library/elasticsearch
eaead16dc43b: Already exists 
e247f12a2b5f: Already exists 
a0495283e709: Already exists 
7c7e458ee599: Already exists 
00ffe59d0e26: Already exists 
aab476f874bd: Already exists 
a20df4c47e64: Already exists 
ef90b430ba5b: Already exists 
79f96f41bfec: Already exists 
Digest: sha256:fdc73b3249c1936f7a277911d58bd10bdd5cf7aae07c1f4d285cf3b7bd18e503
Status: Downloaded newer image for elasticsearch:7.17.8
usage: nc [-46CDdFhklNnrStUuvZz] [-I length] [-i interval] [-M ttl]
          [-m minttl] [-O length] [-P proxy_username] [-p source_port]
          [-q seconds] [-s source] [-T keyword] [-V rtable] [-W recvlimit] [-w timeout]
          [-X proxy_protocol] [-x proxy_address[:port]]           [destination] [port]
```

Reply:
okay perfect, thanks! :)